pandas/tests/test_interval.py

```diff
@@ -63,10 +63,79 @@ def test_hash(self):
         # should not raise
         hash(self.interval)
 
-    # def test_math(self):
-    #     expected = Interval(1, 2)
-    #     actual = self.interval + 1
-    #     self.assertEqual(expected, actual)
+    def test_math_add(self):
+        expected = Interval(1, 2)
+        actual = self.interval + 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(1, 2)
+        actual = 1 + self.interval
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual += 1
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval + Interval(1, 2)
+
+    def test_math_sub(self):
+        expected = Interval(-1, 0)
+        actual = self.interval - 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(-1, 0)
+        actual = 1 - self.interval
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual -= 1
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval - Interval(1, 2)
+
+    def test_math_mult(self):
+        expected = Interval(0, 2)
+        actual = self.interval * 2
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual *= 2
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval * Interval(1, 2)
+
+    def test_math_div_py3(self):
```

Comment:
That's actually what I did originally.  However, I ran into this error.
```bash
pandas/src/interval.pyx:1:31: from __future__ imports must occur at the beginning of the file
building 'pandas.lib' extension
gcc -fno-strict-aliasing -I/Users/mortonjt/miniconda2/envs/pd2/include -arch x86_64 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Ipandas/src/klib -Ipandas/src -I/Users/mortonjt/miniconda2/envs/pd2/lib/python2.7/site-packages/numpy/core/include -I/Users/mortonjt/miniconda2/envs/pd2/include/python2.7 -c pandas/lib.c -o build/temp.macosx-10.5-x86_64-2.7/pandas/lib.o
pandas/lib.c:1:2: error: Do not use this file, it is the result of a failed Cython compilation.
#error Do not use this file, it is the result of a failed Cython compilation.
```
And here's a relevant, unresolved [stackoverflow post](http://stackoverflow.com/questions/28031736/cython-linking-fails-due-to-from-future-import-division-must-occur-at-the)

Which is why I opted to have a separate `__div__` method to handle PY2 division originally.  
What are your thoughts on this?